ros_ign_bridge/package.xml

```diff
@@ -19,6 +19,8 @@
   <depend>rosgraph_msgs</depend>
   <depend>sensor_msgs</depend>
   <depend>std_msgs</depend>
+  <depend>std_srvs</depend>
```

Comment:
Is `std_srvs` needed?

Reply:
It was added in 7b06c21 but it wasn't needed for just that so I removed it. Good call